appsTransport.go

```diff
@@ -20,20 +20,20 @@ import (
 //
 // See https://developer.github.com/apps/building-integrations/setting-up-and-registering-github-apps/about-authentication-options-for-github-apps/
 type AppsTransport struct {
-	BaseURL       string            // BaseURL is the scheme and host for GitHub API, defaults to https://api.github.com
-	Client        Client            // Client to use to refresh tokens, defaults to http.Client with provided transport
-	tr            http.RoundTripper // tr is the underlying roundtripper being wrapped
-	key           *rsa.PrivateKey   // key is the GitHub Integration's private key
-	integrationID int               // integrationID is the GitHub Integration's Installation ID
+	BaseURL string            // BaseURL is the scheme and host for GitHub API, defaults to https://api.github.com
+	Client  Client            // Client to use to refresh tokens, defaults to http.Client with provided transport
+	tr      http.RoundTripper // tr is the underlying roundtripper being wrapped
+	key     *rsa.PrivateKey   // key is the GitHub Integration's private key
+	appID   int               // appID is the GitHub Integration's Installation ID
```

Comment:
Are appID and installationID synonymous then? We said above:

## What is app ID and installation ID
`installation ID` is a part of WebHook request.  
`app ID` is GitHub App ID. \

Reply:
App ID is per application.
Installation ID is per instance of the application.

Done.